docs/spec/consensus/creating-proposal.md

```diff
@@ -0,0 +1,42 @@
+# Creating a proposal
```

Comment:
Awesome thanks for writing this. We should link to it from https://github.com/tendermint/tendermint/tree/develop/docs/spec#consensus-protocol.